apps/issuance/interfaces/issuance.interfaces.ts

```diff
@@ -141,6 +141,15 @@ export interface ICredential{
   '@context':[];
   type: string[];
   prettyVc?: IPrettyVc;
+  issuer?: {
```

Comment:
Any specific reason to use `issuer` as optional parameter ?

Reply:
Yes, we have used that interface at many places, So to ensure that the previous changes work correctly, it has been made optional